### PR TITLE
[tests-only]add tests for deleting files and folder

### PR DESCRIPTION
**Pull request by swoichha**
### Description
Added automated tests for deleting files and folders.

### Related Issues
- Part of https://github.com/owncloud/client/issues/8971#issuecomment-909884844

### CODE REVIEW

File: test/gui/shared/steps/steps.py [hunk 1]
```diff
@@ -806,3 +812,13 @@ def step(context):
     clickButton(
         waitForObject(names.disable_virtual_file_support_Disable_support_QPushButton)
     )
+
+
+@When('the user deletes the file "|any|"')
```
Comment (saw-jan) [inline test/gui/shared/steps/steps.py:817]: can these two be merged?

**Comment by Talank** [comment]
there are conflicting files. Please do rebase.

**Comment by Talank** [comment]
Drone build failed because of `"Error Details": "LookupError: Lost connection to AUT"` . I have restarted the build.

**Review by Talank — APPROVED** [review]
LGTM

**Review by individual-it — APPROVED** [review]

**Pull request merged**